### PR TITLE
Cloud7 tempestrun fixes

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by dirkmueller** [comment]
[(2) #QA: Reintroduce test coverage for heat](https://trello.com/c/0qJztsYh/25-2-qa-reintroduce-test-coverage-for-heat)

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3322,45 +3316,34 @@ function oncontroller_run_tempest
...
-        if tempest help cleanup; then
-            tempest cleanup --init-saved-state
+    local tempestret
+
+    if iscloudver 6plus; then
+        tempest cleanup --init-saved-state
+        local opts=$tempestoptions
+        if iscloudver 7plus; then
+            ### backward compatibility, remove
+            opts=${opts/-s/--smoke}
```
Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: This results in `--smokemoke` if `tempestoptions` included `--smoke` already, so it's causing https://trello.com/c/cvIDZ4TA

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: BTW the C7 gating runs didn't catch this because it was run with `tempestoptions="-t -s"`.

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: See https://github.com/SUSE-Cloud/automation/pull/1575

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: (and #1576)

**Pull request merged**